x-pack/plugins/fleet/server/integration_tests/cloud_preconfiguration.test.ts

```diff
@@ -7,7 +7,8 @@
 
 import Path from 'path';
 
-import * as kbnTestServer from '@kbn/core/test_helpers/kbn_server';
+import type { TestElasticsearchUtils, TestKibanaUtils } from '@kbn/core-test-helpers-kbn-server';
```

Comment:
@elastic/fleet I split the imports out into separate methods and/or types after migraitng the `kbn_server` utilities to a package.